### ISSUE TITLE
Add filters to scoreboards 

### ISSUE THREAD

**Issue by ehguille** [moloch--/RootTheBox#482]
I'll tag this as an enhacement. 

It would be great to have a filter option so you can see the points for each user by corporation, box, category or level.

**Comment by eljeffeg** [comment]
This is similar to the discussion https://github.com/moloch--/RootTheBox/issues/161 where I was thinking of org based scoreboards similar to what PicoCTF can do.

**Comment by ehguille** [comment]
Yes, but I belive my request is easier to implement, it could be just a dropdown box where you can filter by corp, box, etc. 

That way I could check the scoreboard for a single box, so I can host a competition based on this box without caring about previous users points.

If I want to host a new competition, everyuser has background points, so now I have to write down how many points each participant has before the competition and substract them from the score after competition, which can be a mess.
